lib/msf/ui/debug.rb

```diff
@@ -145,7 +148,44 @@ def self.datastore(framework, driver)
           content
         )
       rescue StandardError => e
-        section_build_error('Failed to extract Datastore', e)
+        build_section(
+          'Module/Datastore',
+          ERROR_BLURB,
+          section_build_error('Failed to extract Datastore', e)
+        )
+      end
+
+      def self.database_stats(framework)
```

Comment:
Can we add tests for this :+1:

Reply:
If we now have a dedicated database section, is it worth moving the `db_connection_info` information here as well?

Reply:
For context, the existing tests are here: https://github.com/rapid7/metasploit-framework/blob/5863ff7e357af1634f5b370e811d647a9c999158/spec/lib/msf/debug_spec.rb

Reply:
I'll add tests for this. 👍 
About `db_connection_info`, I think moving it into the `self.database_stats` function is a good idea, right below the table 👍